position.go

```diff
@@ -20,3 +20,11 @@ func (t *Terminal) getTermPosition(pos fyne.Position) position {
 	row := int(pos.Y/cell.Height) + 1
 	return position{col, row}
 }
+
+// getPixelPosition converts a terminal position (row and col) to pixel coordinates.
+func (t *Terminal) getPixelPosition(pos position) fyne.Position {
```

Comment:
Oh sorry I missed this before - the word "Pixel" is misleading here because it's fyne coordinates not pixel coordinates.
Realistically the `fyne.Position` is just a `Position` so perhaps this `position`  is a `textPosition`?

Reply:
done :D